### DISCUSSION TITLE
Purpose of `completion`, `DumpCompletionCommand`?

### DISCUSSION THREAD

**Discussion by bilogic** [laravel-zero/laravel-zero#368] [Q&A] [answered]
Hi,

I was wondering if `completion` is removable, and what is it for actually? Thank you.

**Comment by owenvoke** [comment, marked as answer]
I think this has somehow been pulled in automatically from Symfony. The command is for adding shell autocompletion for your command to your Terminal.

You could hide this command from your list by adding the command to [this config section](https://github.com/laravel-zero/laravel-zero/blob/master/config/commands.php#L57) or remove it completely by adding the class [here](https://github.com/laravel-zero/laravel-zero/blob/master/config/commands.php#L76).

**Reply by bilogic** [reply]
Solved my problem. Thanks!

**Reply by nunomaduro** [reply]
@owenvoke Can you commit that to our skeleton?

**Reply by owenvoke** [reply]
@nunomaduro, do you want it removed or just hidden?
